tensorflow/tools/docker/parameterized_docker_build.sh

```diff
@@ -299,6 +317,26 @@ else
   sed "s/^RUN git clone --branch=.* --depth=1/RUN git clone --branch=${TF_DOCKER_BUILD_DEVEL_BRANCH} --depth=1/" \
       "${ORIG_DOCKERFILE}" > "${DOCKERFILE}"
 
+  # Modify the devel Dockerfile to specify the MKL bazel build parameters
+  if [[ "${TF_DOCKER_BUILD_TYPE}" == "mkl" ]]; then
+    if sed -i -e 's/bazel build -c opt /bazel build -c opt --config=mkl /g' "${DOCKERFILE}"
+    then
+      echo "Modified Dockerfile to add MKL support."
+    else
+      die "FAILED to add MKL support to ${DOCKERFILE}."
+    fi
+  fi
+
+  # Modify the devel Dockerfile with bazel build options
+  if [[ ! -z ${TF_BAZEL_BUILD_OPTIONS} ]]; then
+    if sed -i -e "s/--cxxopt=\"-D_GLIBCXX_USE_CXX11_ABI=0\" /--cxxopt=\"-D_GLIBCXX_USE_CXX11_ABI=0\" ${TF_BAZEL_BUILD_OPTIONS} /g" "${DOCKERFILE}"
```

Comment:
Not really familiar with Dockerfiles, but is there a cleaner/less brittle way of modifying the Bazel build options?

Reply:
Same comment as above. I'm just following the pattern that the original authors were using, regardless of elegance. 

Dockerfiles themselves can have variables that can be overridden on the docker build command. For example, if the following line is in a Dockerfile:

`# ARGs are parameterized. Use --build-args to override.
+ARG WHL_DIR=/whl`

then when building the container, the caller can override the default value of WHL_DIR like this:

`docker build --build-arg WHL_DIR=<some path> . -f <docker file>`

That would be more elegant in my opinion.